### DISCUSSION TITLE
Is there an Event for mobile rotation screen?

### DISCUSSION THREAD

**Discussion by Michel7GitHub** [flet-dev/flet#3894] [Q&A] [answered]
### Question

I want to detect the width/height change (page.width & page.height).
page.on_resize doesn't trigger when I rotate the screen of my mobile.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Can you mention it here: https://github.com/flet-dev/flet/issues/2694

**Reply by Michel7GitHub** [reply]
Done
